res/Sectors/M1105/Gzirr!k'l.txt

```diff
@@ -0,0 +1,561 @@
+# Generated by https://travellermap.com
+# 2018-11-05T13:16:28+00:00
+
+# Gzirr!k'l
+# 5,1
+
+# Name: Gzirr!k'l (Kk)
+
+# Milieu: M1105
+
+# Credits: Gzirr!k'l sector was designed by Mark Humphreys.
+
+# Author:    Mark Humphreys
+
+# Subsector A: Eegr!kutuliroo
+# Subsector B: 't!kr Book'n'
+# Subsector C: Kuuk K! Kaaxk
+# Subsector D: K!t Tet R!'k
+# Subsector E: Gaatabe
+# Subsector F: Koogur Uub
+# Subsector G: Tor K!!g!'
+# Subsector H: Utarrikik'kr'r
+# Subsector I: Gaakookeer
+# Subsector J: K!rrat Kturuk
+# Subsector K: Gn'rr!k!ghooghii
+# Subsector L: Kanepak
+# Subsector M: Gerockne
+# Subsector N: Uan
+# Subsector O: Uckemic
+# Subsector P: Qattruxoone
+
+# Alleg: Hf: "Hive FDA Uplift/Observation Site"
+# Alleg: CsHv: "Hiver Client State"
```

Comment:
To be consistent with other sectors, this allegiance name should be "Client State, Hive Federation"

Reply:
Done